src/Pages/ohyeonkwon/Main/Main.js

```diff
@@ -0,0 +1,271 @@
+                      <span id="indexMsg">Wecode | 위코드</span>
+                    </div>
+                  </div>
+                  <div className="storyBox">
+                    <span id="storyBoxTag">스토리</span>
+                    <button type="button">모두보기</button>
+                  </div>
+                  <div className="stoyContainer">
+                    <ul className="storyList">
+                      <li>
+                        <img
+                          alt="follwer profile"
+                          src="//upload.wikimedia.org/wikipedia/commons/thumb/e/ec/Color_icon_pink.svg/300px-Color_icon_pink.svg.png"
+                        ></img>
+                        <div className="storyFollower">
+                          <span id="name">Test 1</span>
+                          <span id="time">30분전</span>
+                        </div>
+                      </li>
+                      <li>
+                        <img
+                          alt="follwer profile"
+                          src="//upload.wikimedia.org/wikipedia/commons/thumb/e/ec/Color_icon_pink.svg/300px-Color_icon_pink.svg.png"
+                        ></img>
+                        <div className="storyFollower">
+                          <span id="name">Test 1</span>
+                          <span id="time">30분전</span>
+                        </div>
+                      </li>
+                      <li>
+                        <img
+                          alt="follwer profile"
+                          src="//upload.wikimedia.org/wikipedia/commons/thumb/e/ec/Color_icon_pink.svg/300px-Color_icon_pink.svg.png"
+                        ></img>
+                        <div className="storyFollower">
+                          <span id="name">Test 1</span>
+                          <span id="time">30분전</span>
+                        </div>
+                      </li>
+                      <li>
+                        <img
+                          alt="follwer profile"
+                          src="//upload.wikimedia.org/wikipedia/commons/thumb/e/ec/Color_icon_pink.svg/300px-Color_icon_pink.svg.png"
+                        ></img>
+                        <div className="storyFollower">
+                          <span id="name">Test 1</span>
+                          <span id="time">30분전</span>
+                        </div>
+                      </li>
+                      <li>
+                        <img
+                          alt="follwer profile"
+                          src="//upload.wikimedia.org/wikipedia/commons/thumb/e/ec/Color_icon_pink.svg/300px-Color_icon_pink.svg.png"
+                        ></img>
+                        <div className="storyFollower">
+                          <span id="name">Test_1</span>
+                          <span id="time">30분전</span>
+                        </div>
+                      </li>
+                    </ul>
```

Comment:
이러한 부분 맵을 사용해서 효율적으로 바꿀 수 있습니다!!
```suggestion
                    <ul className="storyList">
                     {
                       arr.map((ele, idx) => {
                         return (
                            <li>
                              <img
                                  alt="follwer profile"
                                  src="//upload.wikimedia.org/wikipedia/commons/thumb/e/ec/Color_icon_pink.svg/300px- 
       Color_icon_pink.svg.png"
                                />
                                <div className="storyFollower">
                                  <span id="name">Test 1</span>
                                  <span id="time">30분전</span>
                                </div>
                              </li>
                               )
                             });
                       }
                    </ul>
```

Reply:
같은 영역이 여러번 반복되는 코드이기 때문에 map 메서드를 사용해서 반복되는 부분을 줄여볼까요?
그럼 여기서 arr이란 배열은 어디에 정의를 해줘야 할까요?
render() 함수 밑에 const arr = [1, 2, 3, 4];  이런식으로 정의를 해주시고 사용하시면 됩니다!